client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -54,25 +53,30 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 	};
 
 	renderTodo = () => {
-		const { siteUrl } = this.props;
-		const stepsTodo = [
-			SUMMARY_STEPS.JETPACK_CONNECTION,
-			SUMMARY_STEPS.THEME,
-			SUMMARY_STEPS.PAGES,
-			SUMMARY_STEPS.BLOG,
-		];
-		// If we're not connected, we cannot use selectors from `sites/selectors`.
-		const stepLinks = [
-			'/jetpack/connect?url=' + siteUrl,
-			// TODO: update the following with relevant links
-			siteUrl + '/wp-admin/themes.php',
-			siteUrl + '/wp-admin/post-new.php?post_type=page',
-			siteUrl + '/wp-admin/post-new.php',
-		];
+		const { siteUrl, translate } = this.props;
 
-		return map( stepsTodo, ( fieldLabel, fieldIndex ) => (
-			<div key={ fieldIndex } className="steps__summary-entry todo">
-				<a href={ stepLinks[ fieldIndex ] }>{ fieldLabel }</a>
+		const stepsTodo = {
```

Comment:
Making note: We want to ensure the ordering is preserved. An object of objects should be fine [according to 2ality](http://2ality.com/2015/10/property-traversal-order-es6.html#traversing-the-own-keys-of-an-object):

> Property keys are traversed in the following order:
> - First, the keys that are integer indices (what these are is [explained later](http://2ality.com/2015/10/property-traversal-order-es6.html#integer-indices)), in ascending numeric order.
> - Then, all other string keys, in the order in which they were added to the object.
> - Lastly, all symbol keys, in the order in which they were added to the object.

The top-level object has the advantage of providing a nice string for the `key` prop. I would have reached for an array-of-objects but this is a good approach 👍